stripe/build.gradle

```diff
@@ -1,9 +1,17 @@
 apply plugin: 'com.android.library'
 
+configurations {
+    javadocDeps
+}
+
 dependencies {
+    compile 'com.android.support:support-annotations:25.0.1'
```

Comment:
Adding annotations doesn't (AFAIK) increase the size of your binary. They're just good style.